### ISSUE TITLE
Review language code at facebook plugin

### ISSUE THREAD

**Issue by rodfersou** [collective/sc.social.like#122]
It looks like we have a bug there: https://github.com/collective/sc.social.like/pull/121/files#r129295520

**Comment by hvelarde** [comment]
I don't think we have a bug; that's too obvious: every site would be displaying now a wdget in English and that's not the case. probably the error is on the test.

**Comment by rodfersou** [comment]
@hvelarde create a new site, don't change the language, and our plugin is guessing `en_GB` instead of `en_US`.
The test is right, there is a bug, not important bug since the translated words are probably the same, but still it is a bug.
